cmd/engine/main_oci_worker.go

```diff
@@ -377,7 +377,7 @@ func snapshotterFactory(commonRoot string, cfg config.OCIConfig, sm *session.Man
 				grpc.WithDefaultCallOptions(grpc.MaxCallRecvMsgSize(defaults.DefaultMaxRecvMsgSize)),
 				grpc.WithDefaultCallOptions(grpc.MaxCallSendMsgSize(defaults.DefaultMaxSendMsgSize)),
 			}
-			conn, err := grpc.Dial(dialer.DialAddress(address), gopts...)
+			conn, err := grpc.NewClient(dialer.DialAddress(address), gopts...)
```

Comment:
no problem but curious what the reason for this change is

Reply:
nothing big - `grpc.Dial` is just deprecated, my linter/editor got mad about it in one file so I changed them all